modules/admission-webhook-overview.adoc

```diff
@@ -0,0 +1,52 @@
+
+//Future xref - * At the end of the admissions process, a validating admission controller can invoke xref:../architecture/admission-controllers.adoc#validating-admission-webhooks[validating admission webhooks] to make sure an object is configured properly, such as validating affinity labels. If the validation passes, {product-title} schedules the object as configured.
+* At the end of the admissions process, a validating admission controller can invoke validating admission webhooks to make sure an object is configured properly, such as validating affinity labels. If the validation passes, {product-title} schedules the object as configured.
+
+When an API request comes in, a mutating or validating admission controller uses the list of external webhooks in the configuration and calls them in parallel:
+
+* If all of the webhooks approve the request, the admission chain continues.
+
+* If any of the webhooks deny the request, the admission request is denied and the reason for doing so is based on the first webhook denial reason.
+
+* If more than one webhook denies the admission request, only the first will be returned to the user.
+
+* If there is an error encountered when calling a webhook, that request is either denied or the webhook is ignored.
+
+//Future xrefs - The communication between the admission controller and the webhook server needs to be secured using TLS. Generate a CA certificate and use the certificate to sign the server certificate used by your webhook server. The PEM-formatted CA certificate is supplied to the admission controller using a mechanism, such as xref:../authentication/certificates/service-serving-certificate.adoc#service-serving-certificate[Service Serving Certificate Secrets]. The xref:../architecture/admission-controllers.adoc#admission-webhook-create_admission-controllers[webhook creation procedure] later in this document includes steps to achieve this.
+The communication between the admission controller and the webhook server needs to be secured using TLS. Generate a CA certificate and use the certificate to sign the server certificate used by your webhook server. The PEM-formatted CA certificate is supplied to the admission controller using a mechanism, such as Service Serving Certificate Secrets. The webhook creation procedure later in this document includes steps to achieve this.
+
+The following diagram illustrates the sequential admission chain process within which multiple webhook servers are called.
+
+.API admission chain with mutating and validating webhooks
+image::api-admission-chain.png["API admission stage", align="center"]
+
+A simple example use case for admission webhooks is syntactical validation of resources. For example, in an infrastructure where all pods must have a common set of labels, a webhook could inject labels and another webhook could verify that labels are as expected. {product-title} would subsequently schedule pods which include required labels and reject those which do not.
```

Comment:
* A simple example -> An example
* could -> can (in two places)
* which include required labels and reject those which do not. -> that include the required labels and reject those that do not.

Reply:
Each of these points now resolved.